src/main/java/Teste.java

```diff
@@ -0,0 +1,22 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Teste {
```

Comment:
What is this class for ?

Reply:
that  was not supposed to  be pushed, already removed